tailwind.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     extend: {
       fontFamily: {
         display: ['Lato', 'sans-serif'],
-        body: ['Noto sans', 'sans-serif'],
+        body: ['"Noto sans"', 'sans-serif'],
```

Comment:
Extra quotes here

Reply:
I'm used to always add double quotes to font-family that have multiple words.

Example from Taillwind base css:
![image](https://user-images.githubusercontent.com/114004123/205361439-29a5bbe7-a745-4a3b-952f-4112ca44584e.png)

Reply:
Note: If a font name contains white-space, it must be quoted.
https://www.w3schools.com/cssref/pr_font_font-family.php#:~:text=If%20a%20font%20name%20contains%20white%2Dspace%2C%20it%20must%20be%20quoted

Reply:
The more your know 😄 Just checked Tailwind's documentation and they're doing the same thing here: https://tailwindcss.com/docs/font-family#customizing-your-theme so I'm good with this.